src/components/dashboard/DashboardPage.js

```diff
@@ -7,6 +8,13 @@ import Fab from "@material-ui/core/Fab";
 import Typography from "@material-ui/core/Typography";
 import Tooltip from "@material-ui/core/Tooltip";
 import HackathonCard from "./HackathonCard";
+import { CREATE_HACKATHON_ROUTE } from "../../routes";
+
+/** The routes that we might redirect to by clicking a button. */
+const REDIRECT = {
+  NONE: "",
+  CREATE: <Redirect to={CREATE_HACKATHON_ROUTE} />
+};
```

Comment:
Cool, I didn't know that! Right now, I'm not passing any props in for this particular route—should I use that convention anyways (since I will be doing that for routes down the road)?